lib/instrumentation/span.js

```diff
@@ -36,6 +36,7 @@ function Span (transaction, name, ...args) {
 
   this.transaction = transaction
   this.name = name || 'unnamed'
+  this.outcome = 'unknown'
```

Comment:
can you move this to `GenericsSpan` class as `outcome` belongs to both Transaction and Span.